lib/hocon/cli.rb

```diff
@@ -0,0 +1,186 @@
+require 'optparse'
+require 'hocon'
+require 'hocon/config_render_options'
+require 'hocon/config_factory'
+require 'hocon/config_value_factory'
+require 'hocon/parser/config_document_factory'
+require 'hocon/config_error'
+
+module Hocon::CLI
+  # Aliases
+  ConfigMissingError = Hocon::ConfigError::ConfigMissingError
+  ConfigWrongTypeError = Hocon::ConfigError::ConfigWrongTypeError
+
+  # List of valid subcommands
+  SUBCOMMANDS = ['get', 'set', 'unset']
+
+  # Parses the command line flags and argument
+  # Returns a options hash with values for each option and argument
+  def self.parse_args(args)
+    options = {}
+    opt_parser = OptionParser.new do |opts|
+      subcommands = SUBCOMMANDS.join(',')
+      opts.banner = "Usage: hocon [--file HOCON_FILE] {#{subcommands}} PATH [VALUE]"
```

Comment:
Yes that would be helpful. So is there no longer a way to specify just one file (which will be both read in and then written to) when attempting to modify a variable?